Bulldozer.java

```diff
@@ -0,0 +1,10 @@
+public class Bulldozer extends Machine {
```

Comment:
where si `package core.basesyntax;`? the same for other classes

Reply:
What is core.basesyntax; ? Where can I read about that? Should I import this package, or it is about doing syntax right?